lib/shopify_api/session.rb

```diff
@@ -143,6 +144,16 @@ def api_version=(version)
       end
     end
 
+    def access_scopes=(access_scopes)
```

Comment:
Does this need to be public?

Reply:
yeah this shouldn't be public

Reply:
Yeah I guess I meant that it's public right now and we could probably make it private/protected.